en/firefox/accounts.ftl

```diff
@@ -5,74 +5,46 @@
 ### URL: https://www-dev.allizom.org/firefox/accounts/
 
 # HTML page title
-firefox-accounts-there-is-a-way-to = There is a way to protect your privacy. Join { -brand-name-firefox }.
-
-# Old HTML page title
 firefox-accounts-get-a-firefox-account = Get a { -brand-name-firefox-account } – Keep your data private, safe and synced
 
 # HTML page description
-firefox-accounts-take-your-stand-stay-smart = Take your stand against an industry that’s selling your data to third parties. Stay smart and safe online with technology that fights for you.
-
-# Old HTML page description
 firefox-accounts-securely-sync-your = Securely sync your passwords, bookmarks and tabs across all your devices. Get a { -brand-name-firefox-account } now – One login – Power and privacy everywhere.
 
-# The span here is for visual formatting to display the phrase "Join Firefox" in a different color.
-firefox-accounts-there-is-a-way-to-protect = There is a way to protect your privacy. <span>Join { -brand-name-firefox }.</span>
-
-# This refers to the way some tech companies treat personal information as a commodity, as if their users are the product they're selling to their advertisers.
-firefox-accounts-take-your-stand-against = Take your stand against an industry that’s making you the product.
-
-# The span here is to display the phrase "Now try Firefox Monitor." in a different color. Line breaks are for visual formatting.
-firefox-accounts-youre-signed-in-to = You’re signed <br>in to { -brand-name-firefox }. <br><span>Now try { -brand-name-firefox-monitor }.</span>
-
-firefox-accounts-see-if-youve-been = See if you’ve been involved in an online data breach.
-firefox-accounts-sign-in-to-monitor = Sign In to { -brand-name-monitor }
-firefox-accounts-join-firefox = Join { -brand-name-firefox }
 firefox-accounts-enter-your-email-address = Enter your email address to get started.
 firefox-accounts-already-have-an-account = Already have an account?
 firefox-accounts-sign-in = Sign In
+firefox-accounts-already = You already have a { -brand-name-firefox-account }. Congrats!
+firefox-accounts-manage = Manage your account
+
+firefox-accounts-sign-in-to = Sign in to your { -brand-name-firefox-account } to:
```

Comment:
What's the context following this string?

Reply:
It's a list of different things you can do with a Firefox account. We should add a comment.